lib/gcs.rb

```diff
@@ -30,6 +30,36 @@ def buckets(project_id)
     @api.list_buckets(project_id, max_results: 1000).items || []
   end
 
+  def bucket(name)
+    @api.get_bucket(name)
+  rescue Google::Apis::ClientError
+    if $!.status_code == 404
+      return nil
+    else
+      raise
+    end
+  end
+
+  def insert_bucket(project_id, name, storage_class: "MULTI_REGIONAL", acl: nil, default_object_acl: nil, location: nil)
+    b = Bucket.new(
+      name: name,
+      location: location,
```

Comment:
Missing `storage_class`, Is it intended?
https://github.com/google/google-api-ruby-client/blob/0.14.5/generated/google/apis/storage_v1/classes.rb#L119-L127